UnityGsdk/Assets/PlayFabSdk/MultiplayerAgent/Helpers/ISerializer.cs

```diff
@@ -78,6 +78,31 @@ protected override bool TrySerializeKnownTypes(object input, out object output)
                 }
                 return base.TrySerializeKnownTypes(input, out output);
             }
+            
+            public static readonly string[] _defaultDateTimeFormats = new string[]{ // All parseable ISO 8601 formats for DateTime.[Try]ParseExact - Lets us deserialize any legacy timestamps in one of these formats
```

Comment:
>_defaultDateTimeFormats [](start = 44, length = 23)

Very minor: Our naming convention for "static readonly" is "DefaultDateTimeFormats"

Reply:
fixing